src/main/java/com/infoshareacademy/service/FridgeService.java

```diff
@@ -47,8 +50,17 @@ public Optional<Fridge> findFridgeById(Long id) {
         return fridgeRepository.findById(id);
     }
 
-
     public long getUserId() {
         return userId;
     }
+
+    public ProductInFridge findProductInFridgeById(Long productId) throws Exception {
+        return productInFridgeRepository.findById(productId)
+                .orElseThrow(() -> new Exception("Not found Product in Fridge for"
+                        + "ID: " + productId));
+    }
```

Comment:
```suggestion
        public ProductInFridge findProductInFridgeById(Long productId) throws NotFoundException {
        return productInFridgeRepository.findById(productId)
                .orElseThrow(() -> new NotFoundException(MessageFormat.format("Not found Product in Fridge for ID: {}", productId)));
    }
```
Cieszy mnie widok takiego kodu, używanie optionali i rzucanie wyjątków jest dobrą drogą. 
Niestety są dwie rzeczy które muszę się przyczepic. 
1. Jeśli nie znajdziemy zasobu który szukamy/sprawdzamy rzucamy NotFoundException
2. Optymalniejszą opcją konkatenacji strinfow jest używanie MessageFormatera, staraj sie nie łączyć ich za pomocą "+"
3. Napsałaś metode której nigdzie nie wykorzystujesz